### DISCUSSION TITLE
Settings not showing after installing

### DISCUSSION THREAD

**Discussion by xinzihou** [SilentVoid13/Templater#1102] [Help]
Hi everyone,

I've been using Templater for about a year and suddenly Alt+E didn't work. Soon I realised that there was no _Templater_ under the tab _Plugin options_. I used the in-built _Template_ plugin in Obsidian to insert my template, but the automatic fill-in fields didn't work either. 
<img width="338" alt="image" src="https://user-images.githubusercontent.com/97024733/236570506-3e7dee84-1aac-4786-a179-be519704846e.png">

Before the issue appeared, the only things I did were creating a new vault and installing the _Dictionary_ plugin.  I tried restarting Obsidian several times and also rebooting my MacBook, but none of them worked. Could anyone help me to find the cause of this issue please? Thank you so much!

Templater installed, but no setting options in the tool bar on the left:
<img width="902" alt="image" src="https://user-images.githubusercontent.com/97024733/236569826-d70a485b-97e1-4f3d-86ff-3e8a3152eb0e.png">

---
Update:
I downgraded the plugin to 1.12.0 (which I think should be the version I used before updating) and it finally works... I'm using Obsidian v0.14.6 and it's the latest version for my device. Hopefully the problem with the latest plugin could be solved!

**Comment by Zachatoo** [comment]
I have no idea why the Templater plugin would do that, so I can't be much help there.

However, you definitely aren't on the latest version of Obsidian, that's gotta be about 1-2 years old. The latest version should be v1.2.8. I highly recommend upgrading to ensure other plugins, including Templater, continue to work as you update them.

<img width="1070" alt="image" src="https://user-images.githubusercontent.com/6936914/236685592-197f3b24-ea58-4e7a-83a1-93736f040b04.png">

If you can't update from within Obsidian, you should be able to download a new installer from the Obsidian website and install over your current Obsidian copy, no need to uninstall first.

I will say, that v1.x.x of Obsidian does change the look and feel of Obsidian in quite a few ways, some older themes may not work once you upgrade if the theme maintainer hasn't pushed an update to match v1.x.x styling. And it's a pain to downgrade your Obsidian version.

**Reply by xinzihou** [reply]
Thank you so much! I tried to update everything in Obsidian (including Obsidian) and now it works fine.
